packages/karma/karma.conf.js

```diff
@@ -188,7 +188,7 @@ try {
    */
   function configureFiles(conf) {
     overrideConfigValue(conf, 'files', [
-      TMPL_bootstrap_files
+      TMPL_bootstrap_files,
```

Comment:
The comma is not necessary because every line is appended with a comma when this template is appended.
See https://github.com/bazelbuild/rules_nodejs/blob/master/packages/karma/karma_web_test.bzl#L168